src/positions/types.ts

```diff
@@ -46,3 +53,17 @@ export interface ContractPosition extends AbstractPosition {
 
 export type Token = BaseToken | AppTokenPosition
 export type Position = AppTokenPosition | ContractPosition
+
+export interface Shortcut {
+  id: string
+  appId: string
+  name: string
+  description: string
+  networks: string[]
+  category?: 'claim'
+}
+
+export interface ClaimablePosition extends Omit<Position, 'availableShortcutIds' | 'tokens'> {
+  shortcutId: string
+  claimableTokens: TokenWithCategory[]
```

Comment:
i think this type makes it easier to display the claimable positions, but maybe there's a more elegant way to do this. since a position can have multiple "claim" shortcuts, and we need to display each "claim" shortcut as an item on the UI, my idea is to have a list of positions where each position has a "claim" shortcut and a list of claimable tokens. 

note that the claimable tokens can have nested tokens but the UI should only need to look on the top level for which tokens to display, so the selector moves all the nested claimable tokens to the top level

Reply:
This looks good to me 👍 

I wonder if we want a direct item type which contains both the shortcuts info + claimableTokens, directly consumable by the list component that will display them?

Reply:
yeah that's a good idea. i updated, is this what you were thinking?